airbyte-webapp/src/components/PageTitle/PageTitle.tsx

```diff
@@ -11,56 +13,26 @@ interface PageTitleProps {
   title: React.ReactNode;
 }
 
-export const MainContainer = styled.div<{ withLine?: boolean }>`
-  padding: 20px 32px 18px;
-  border-bottom: ${({ theme, withLine }) => (withLine ? `1px solid ${theme.greyColor20}` : "none")};
-  position: relative;
-  z-index: 2;
-  color: ${({ theme }) => theme.darkPrimaryColor};
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  align-items: center;
-`;
-
-export const MiddleBlock = styled.div`
-  flex: 1 0 0;
-  display: flex;
-  justify-content: center;
-`;
-
-export const MiddleTitleBlock = styled(H3)`
-  flex: 1 0 0;
-  display: flex;
-  justify-content: center;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-`;
-
-export const EndBlock = styled.div`
-  flex: 1 0 0;
-  display: flex;
-  justify-content: flex-end;
-`;
-
-export const TitleBlock = styled(H3)`
-  flex: 1 0 0;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-`;
-
 const PageTitle: React.FC<PageTitleProps> = ({ title, withLine, middleComponent, middleTitleBlock, endComponent }) => (
```

Comment:
This component should be named `PageHeader` and moved to `src/components/ui/`
Can you also add a storybook file?

Reply:
@edmundito added a storybook file with a `Primary` story. I wasn't sure of the best way to implement the 'middle' component in a story as it needs to hold state, if that needs to be included in this PR - happy to do it, but need more direction :)